diff --unified --recursive --text osTicket/include/class.plugin.php osTicket.new/include/class.plugin.php
--- osTicket/include/class.plugin.php	2022-02-16 14:41:13.641451932 +0100
+++ osTicket.new/include/class.plugin.php	2022-02-16 15:25:52.498200904 +0100
@@ -251,7 +251,7 @@
      */
     static function allInfos() {
         foreach (glob(INCLUDE_DIR . 'plugins/*',
-                GLOB_NOSORT|GLOB_BRACE) as $p) {
+                GLOB_NOSORT) as $p) {
             $is_phar = false;
             if (substr($p, strlen($p) - 5) == '.phar'
                     && class_exists('Phar')
